14_2_elements.js

```diff
@@ -0,0 +1,12 @@
+function byTagName(node, tagName) {
+    const elements = node.querySelectorAll(tagName);
+    return [...elements].map(el => el.nodeName.toLowerCase());
+}
+
+console.log(byTagName(document.body, 'h1').length);
+// → 1
+console.log(byTagName(document.body, 'span').length);
+// // → 3
+let para = document.querySelector('p');
```

Comment:
You can use `const` here. And, what `para` means? If it is `paragraph`, then please, use full name to omit any misunderstandings.